<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alt_usuarios</class>
 <widget class="QWidget" name="alt_usuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>330</y>
     <width>350</width>
     <height>481</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="l_nombre">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>129</y>
      <width>321</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_acceso">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>331</width>
      <height>40</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Admin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>User</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="le_password2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>334</y>
      <width>331</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="l_password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>215</y>
      <width>321</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_password2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>304</y>
      <width>331</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Repetir Contraseña</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_nivelacceso">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>395</y>
      <width>321</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nivel de Acceso</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>246</y>
      <width>331</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nombre">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>159</y>
      <width>331</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancelar">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>10</y>
      <width>112</width>
      <height>108</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../template/black/icono_cancelar.png</normaloff>../template/black/icono_cancelar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ayuda">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>112</width>
      <height>108</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../template/black/ayuda.png</normaloff>../template/black/ayuda.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aceptar">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>10</y>
      <width>112</width>
      <height>108</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../template/black/icono_aceptar.png</normaloff>../template/black/icono_aceptar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="fondo_sombra">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>790</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_logo">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>390</y>
     <width>451</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../logo/logo_empresa.png</pixmap>
   </property>
  </widget>
  <zorder>l_logo</zorder>
  <zorder>fondo_sombra</zorder>
  <zorder>frame</zorder>
 </widget>
 <tabstops>
  <tabstop>le_nombre</tabstop>
  <tabstop>le_password</tabstop>
  <tabstop>le_password2</tabstop>
  <tabstop>cb_acceso</tabstop>
  <tabstop>btn_ayuda</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
